lib/lightning_web/live/workflow_live/edit.ex

```diff
@@ -517,9 +517,9 @@ defmodule LightningWeb.WorkflowLive.Edit do
                           phx-value-id={@selected_job.id}
                           theme="danger"
                           disabled={
-                            (!is_nil(@workflow.deleted_at) or !@can_edit_workflow or
-                               @has_child_edges or @is_first_job or
-                               @snapshot_version_tag != "latest") ||
+                            !is_nil(@workflow.deleted_at) or !@can_edit_workflow or
+                              @has_child_edges or @is_first_job or
+                              @snapshot_version_tag != "latest" ||
```

Comment:
I think it's time we start extracting these disabled logic into functions for maintainability. But that is out of scope of this work. Just wondering if we should have an issue to track it.